.github/workflows/test.yaml

```diff
@@ -10,18 +10,18 @@ jobs:
   test:
     strategy:
       matrix:
-        os: 
+        os:
           - ubuntu-20.04
           - ubuntu-22.04
           - windows-2019
           - windows-2022
-          - macOS-12
-          - macOS-11
+          - macOS-14
+          - macOS-13
     runs-on: ${{ matrix.os }}
-    
+
     steps:
     - name: set just version
-      run: echo "JUST_VERSION=1.11.0" >> $GITHUB_ENV
+      run: echo "JUST_VERSION=1.34.0" >> $GITHUB_ENV
```

Comment:
Can we make this a top level env thing?
```
env:
    JUST_VERSION: 1.34.0
```

and then just use $JUST_VERSION in the step bodies?

Reply:
Great idea, just added that